checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py

```diff
@@ -41,7 +52,14 @@ def evaluate_terraform(input_str: Any, keep_interpolations: bool = True) -> Any:
     evaluated_value = evaluate_json_types(evaluated_value)
     second_evaluated_value = _try_evaluate(evaluated_value)
 
-    return evaluated_value if callable(second_evaluated_value) else second_evaluated_value
+    # return evaluated_value if callable(second_evaluated_value) else second_evaluated_value
```

Comment:
```suggestion
```
feel free to remove the commented code block 🙂

Reply:
Oops. thanks